_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -158,7 +158,7 @@ Our subsetting operation reduces the `features` count to 2. This means
 that only two feature lines in our spatial object have the attribute
 `TYPE == footpath`. We can plot only the footpath lines:
 
-```{r plot-subset-shapefile}
+```{r plot-subset-shapefile fig.cap="Map of the footpaths in the study area."}
```

Comment:
comma comma comma comma chameleon

```suggestion
```{r plot-subset-shapefile, fig.cap="Map of the footpaths in the study area."}
```

Reply:
🤦‍♀️ Thanks @zkamvar!